.github/workflows/reusable-linters.yml

```diff
@@ -21,7 +21,7 @@ env:
   PY_COLORS: 1  # Recognized by the `py` package, dependency of `pytest`
   PYTHONIOENCODING: utf-8
   PYTHONUTF8: 1
-  PYTHON_LATEST: 3.12
+  PYTHON_LATEST: 3.13
```

Comment:
Could you move changes that aren't directly related into separate PRs? This would make the diff smaller and easier to follow in general.